### DISCUSSION TITLE
General install instruction doesn't work on Ubuntu

### DISCUSSION THREAD

**Discussion by gokhannY** [gnudatalanguage/gdl#1680] [General]
Hey all,

I've been trying to install gdl on my Ubuntu 22.04 x64 laptop, not that it fails but even simple commands, such as exit, doesn't work.

I use simple installation via sudo apt install gnudatalanguage. What might be the reason for this behaviour? Do I do something wrong via installation or do I need external packages for even simple commands? I have attached screenshots of command line.

Thanks in advance
![ss](https://github.com/gnudatalanguage/gdl/assets/151953408/3fb729b1-92f0-4abf-807a-7131afe2542e)

**Comment by GillesDuvert** [comment]
Hello @gokhannY ,
GDL says 'Procedure not found: EXiT'
it should say 'Procedure not found: EXIT'
There could be a problem with the "localisation" or /and character encoding. Unfortunately, GDL works only with ASCII  ISO-8859-1 . 
BTW, version 1.0.1 on Ubuntu is reported not working. We are at 1.0.3 (there is an 1.0.3 on Ubuntu Kinetic according to [GDL's Wiki](https://github.com/gnudatalanguage/gdl/wiki/GDL-on-Linux)) and a 1.0.4 is due in the next days. The weekly builds may be handy in this case.

**Reply by gokhannY** [reply]
Thanks for quick answer.
I had no problem with prior GDLs or Ubuntu versions with localization. That's weird but I will try to compile weekly builds as you suggested. I thought these builds are automatically updated via package manager. I guess these weekly builds are not official builds.
I will report back as soon as possible.
